crates/router/src/connector/rapyd.rs

```diff
@@ -820,8 +827,32 @@ impl api::IncomingWebhook for Rapyd {
             .body
             .parse_struct("RapydIncomingWebhook")
             .change_context(errors::ConnectorError::WebhookEventTypeNotFound)?;
-
-        webhook.webhook_type.try_into()
+        Ok(match webhook.webhook_type {
+            rapyd::RapydWebhookObjectEventType::PaymentCompleted
+            | rapyd::RapydWebhookObjectEventType::PaymentCaptured => {
```

Comment:
why do we send `Success` status for captured event. Don't send any webhook for Non terminal status
```suggestion
 => {
```